demo/demoapp/views.py

```diff
@@ -2,6 +2,8 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render, redirect
 from djgentelella.notification import create_notification
+from .forms import RawFooForm, FooForm
```

Comment:
Better name for RawFooForm is required

Reply:
changed to FooModelForm and FooBasicForm